crates/store/re_chunk_store/src/writes.rs

```diff
@@ -37,6 +37,14 @@ impl ChunkStore {
             return Ok(Vec::new());
         }
 
+        #[cfg(debug_assertions)]
+        for (component_name, per_desc) in chunk.components().iter() {
+            assert!(
+                per_desc.len() <= 1,
+                "Insert Chunk with multiple values for component named `{component_name}`: this is currently UB",
+            );
+        }
```

Comment:
One day this will be legal, but first we need to remove every single bit of component-name based logic across the entire app.